src/App.jsx

```diff
@@ -1,21 +1,19 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
 import './App.css'
+import TodoItem from './components/TodoItem'
 
 function App() {
   const [count, setCount] = useState(0)
 
   return (
-
-
     <div className="App">
```

Comment:
As a kindly suggestion, keep in mind the the todo app logic, including the InputTodo and the TodoList should be isolated on their own container component called TodosLogic.